.github/workflows/tests-unit-php.yml

```diff
@@ -122,29 +122,6 @@ jobs:
         with:
           composer-options: '--prefer-dist --no-progress --no-interaction --no-scripts'
 
-      # Installs a different PHPUnit version depending on the WP/PHP version combo we're testing against.
-      #
-      # | WP  / PHP | PHPUnit |
-      # |-----------|---------|
-      # | 5.7 / 7.4 | 7       |
-      # | 5.9 / 7.4 | 8       |
-      # | 5.9 / 7.4 | 9       |
-      # | *   / 8   | 9       |
-      #
-      # See https://make.wordpress.org/core/handbook/references/phpunit-compatibility-and-wordpress-versions/
-      - name: Update PHPUnit
-        run: |
-          if [[ $WP_VERSION == "5.7" && $PHP_VERSION == "7.4" ]]; then
-            echo "Installing PHPUnit 7.5.x"
-            composer require --no-interaction --no-scripts phpunit/phpunit:^7.5 --with-dependencies
-          elif [[ $WP_VERSION == "latest" || $WP_VERSION == "trunk" || $PHP_VERSION == "7.4" ]]; then
-            echo "Installing latest version of PHPUnit"
-            composer update --ignore-platform-reqs --no-interaction --no-scripts yoast/phpunit-polyfills --with-dependencies
-          fi
-        env:
-          WP_VERSION: ${{ matrix.wp }}
-          PHP_VERSION: ${{ matrix.php }}
-
```

Comment:
Right now, with removing this, all tests are running with PHPUnit 8.5 as can be seen here:

https://github.com/GoogleForCreators/web-stories-wp/actions/runs/3378530306/jobs/5608853638#step:16:16

We still need the `composer update --ignore-platform-reqs --no-interaction --no-scripts yoast/phpunit-polyfills --with-dependencies` line to update to PHPUnit 9.5.

Once we actually bump the minimum PHP version requirement in #12583, we can remove this and update the dependency permanently in `composer.json` via `composer update`